### PR TITLE
golang: bump to 1.13.0

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

Let's see how many things brake...

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/9118)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-missed-k8s

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

File: Makefile [hunk 1]
```diff
@@ -55,6 +55,23 @@ TEST_UNITTEST_LDFLAGS= -ldflags "-X github.com/cilium/cilium/pkg/kvstore.consulD
 	-X github.com/cilium/cilium/pkg/datapath.DatapathSHA=1234567890abcdef7890 \
 	-X github.com/cilium/cilium/pkg/logging.DefaultLogLevelStr=$(LOGLEVEL)"
 
+define generate_k8s_api
```
Comment (vadorovsky) [inline Makefile:58]: That's a good refactor :+1:

File: daemon/state.go [hunk 2]
```diff
@@ -158,7 +158,7 @@ func (d *Daemon) restoreOldEndpoints(dir string, clean bool) (*endpointRestoreSt
 			continue
 		}
 
-		scopedLog.Debug("Restoring endpoint")
+		scopedLog.Debugf("aanm Restoring endpoint %+v", ep)
```
Comment (vadorovsky) [inline daemon/state.go]: Remove your nickname ;)

File: pkg/endpoint/restore.go [hunk 3]
```diff
@@ -140,6 +140,7 @@ func (e *Endpoint) restoreIdentity() error {
 	scopedLog := log.WithField(logfields.EndpointID, e.ID)
 	// Filter the restored labels with the new daemon's filter
 	l, _ := labels.FilterLabels(e.OpLabels.AllLabels())
+	scopedLog.Debugf("aanm e.OpLabels %+v", e.OpLabels)
```
Comment (vadorovsky) [inline pkg/endpoint/restore.go]: Same here

**Review by vadorovsky — CHANGES_REQUESTED** [review]

File: daemon/state.go (hunk 2 above)
Comment (aanm) [reply daemon/state.go]: @mrostecki the PR is still wip ;-)

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
never-tell-me-the-odds

File: Dockerfile.builder [hunk 4]
```diff
@@ -14,7 +14,7 @@ WORKDIR /go/src/github.com/cilium/cilium
 ENV GOROOT /usr/local/go
 ENV GOPATH /go
 ENV PATH "$GOROOT/bin:$GOPATH/bin:$PATH"
-ENV GO_VERSION 1.12.8
+ENV GO_VERSION 1.13
```
Comment (ianvernon) [inline Dockerfile.builder:17]: Is there a reason this is 1.13 vs. 1.13.0?

Comment (aanm) [reply Dockerfile.builder:17]: I'll see you in the 1.14 release

https://github.com/cilium/cilium/pull/7211#discussion_r261011298

**Review by ianvernon — APPROVED** [review]

**Review by vadorovsky — APPROVED** [review]

**Pull request merged**